experiments/generic/pod-delete/pod-delete.yml

```diff
@@ -0,0 +1,64 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: pod-delete
+spec:
+  template:
+    metadata:
+      labels:
+        app: pod-delete
+    spec:
+      #nodeName:
+        ## mandatory: provide name of host where target container resides
+        ## kubernetes.io/hostname: ""
+      restartPolicy: Never
+      containers:
+      - image: litmuschaos/pod-deleter:latest
+        imagePullPolicy: Always
+        name: pod-delete
+        command: 
+        - /bin/sh
+        args:
+        - ./pod-delete.sh 
```

Comment:
`./pod-delete.sh ` should'nt be go binary?

Reply:
It is not updated yet.. I will update this file